core/shared/src/main/scala/zio/stm/ZSTM.scala

```diff
@@ -684,11 +684,30 @@ object ZSTM {
     }
 
   /**
-   * Creates an STM effect from a `Try` value.
+   * Lifts a function `R => A` into a `ZSTM[R, Nothing, A]`.
+   */
+  def fromFunction[R, A](f: R => A): ZSTM[R, Nothing, A] =
```

Comment:
Could also implement this and `fromFunctionM` as `access(f)` and `assessM(f)`.

Reply:
Good point. Should we change that in ZIO as well?

Reply:
Probably slightly more efficient since we do it in one `Read` operation` instead of a `Read` and a `FlatMap`.

Reply:
Done.